papers/IDML/data/BANKING77/labels.txt

```diff
@@ -0,0 +1,77 @@
+transfer_not_received_by_recipient
```

Comment:
Should all these files under /data be here?

Reply:
Yes.